### PR TITLE
Add Guild JSdocs

### PR DESCRIPTION
**Pull request by brandonbothell**
I said I wasn't done, I thought you would leave it open and wait for the rest. I still have Referrals and Users to go.

### CODE REVIEW

**Comment by tgoins** [comment]
Oh, woops.

**Comment by brandonbothell** [comment]
I'll add the User docs in a whole other thing, because those are going to be something to tackle through. They'll be bunched together with another PR down the road.

@tannerz28 done.

File: src/entity/Guild/GuildSuggestion.ts [hunk 1]
```diff
@@ -4,36 +4,91 @@ import { IsString, IsDate, IsNumber } from 'class-validator'
...
+   *
+   * @type {Date}
+   * @memberof GuildSuggestion
+   */
   @Column()
   @IsDate()
   dateCreated: Date
 
+  /**
+   * The amount of likes/thumbs up the suggestion has gotten.
```
Comment (tgoins) [inline src/entity/Guild/GuildSuggestion.ts]: Changed `gotten` to `received`

File: src/entity/Guild/GuildSuggestion.ts [hunk 2]
```diff
@@ -4,36 +4,91 @@ import { IsString, IsDate, IsNumber } from 'class-validator'
...
+   *
+   * @type {number}
+   * @memberof GuildSuggestion
+   */
   @Column()
   @IsNumber()
   likes: number
 
+  /**
+   * The amount of dislikes/thumbs down the suggestion has gotten.
```
Comment (tgoins) [inline src/entity/Guild/GuildSuggestion.ts]: Changed `gotten` to `received`

**Review by tgoins — CHANGES_REQUESTED** [review]

**Review by tgoins — APPROVED** [review]

**Pull request merged**